build.assets/tooling/cmd/buf-plugin-linters/default.go

```diff
@@ -0,0 +1,331 @@
+			},
+
+			"teleport.lib.teleterm.v1.TerminalService.GetAccessRequests": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+				name: true,
+			},
+
+			"teleport.lib.teleterm.v1.TerminalService.GetRequestableRoles": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+				name: true,
+			},
+
+			"teleport.lib.teleterm.v1.TerminalService.ListKubernetesResources": {
+				skip:  true, // TODO(okraport): Keys are reserved, add pagination and remove skip.
+				size:  "limit",
+				token: "next_key",
+				next:  "next_key",
+			},
+			"teleport.lib.teleterm.v1.TerminalService.ListDatabaseServers": {
+				skip: true, // TODO(okraport): add pagination size limit and token to request and remove skip.
+				next: "next_key",
+			},
+			"teleport.lib.teleterm.v1.TerminalService.ListUnifiedResources": {
+				size:  "limit",
+				token: "start_key",
+				next:  "next_key",
+			},
+			"teleport.lib.teleterm.v1.TerminalService.ListGateways": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+			},
+
+			// GetServiceInfoResponse does not require pagination.
+			"teleport.lib.teleterm.vnet.v1.VnetService.GetServiceInfo": {skip: true},
+
+			// AuthSettings contains repeated `auth_providers` but does not need pagination.
+			"teleport.lib.teleterm.v1.TerminalService.GetAuthSettings": {skip: true},
+
+			"teleport.lib.teleterm.v1.TerminalService.ListRootClusters": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+			},
+			"teleport.lib.teleterm.v1.TerminalService.ListLeafClusters": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+			},
+			"teleport.lib.teleterm.v1.TerminalService.ListDatabaseUsers": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+			},
+
+			"proto.AuthService.GetInventoryStatus": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+			},
+
+			// repeated field `schemas` in `Resource` does not require pagination.
+			"teleport.scim.v1.SCIMService.GetSCIMResource":    {skip: true},
+			"teleport.scim.v1.SCIMService.CreateSCIMResource": {skip: true},
+			"teleport.scim.v1.SCIMService.UpdateSCIMResource": {skip: true},
+
+			// `Device` message contains repeated field `DeviceCollectedData` but is not paginated.
+			"teleport.devicetrust.v1.DeviceTrustService.CreateDevice": {skip: true},
+			"teleport.devicetrust.v1.DeviceTrustService.UpdateDevice": {skip: true},
+			"teleport.devicetrust.v1.DeviceTrustService.UpsertDevice": {skip: true},
+			"teleport.devicetrust.v1.DeviceTrustService.GetDevice":    {skip: true},
+
+			"teleport.lib.vnet.v1.ClientApplicationService.SessionSSHConfig": {skip: true},
+
+			"teleport.devicetrust.v1.DeviceTrustService.FindDevices": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+			},
+			"teleport.devicetrust.v1.DeviceTrustService.BulkCreateDevices": {
+				skip: true, // TODO(okraport): add pagination and remove skip.
+			},
+
+			// Legacy APIs:
```

Comment:
Are these all RPCs that have any repeated field in the response? Why is `GenerateDatabaseCert` in the list?